docs/explanations/what-are-dimensions.rst

```diff
@@ -3,11 +3,11 @@
 What are Dimensions?
 ====================
 
-If a Spec tells you the parameters of a scan, Dimensions gives you the positions
+If a Spec tells you the parameters of a scan, Dimensions gives you the points
 that will let you actually exectute the scan. A list of Dimensions is
 interpreted as nested from slowest moving to fastest moving, so each faster
 Dimension will iterate once per position of the slower Dimension. When
-fly-scanning they key will traverse lower-position-upper on the fastest
+fly-scanning the axis will traverse lower-midpoints-upper on the fastest
```

Comment:
```suggestion
fly-scanning the axis will traverse lower-midpoint-upper on the fastest
```

Reply:
Updated. Also found and resolved a duplication of lower-midpoints-upper in technical terms and core.py